FirebaseAdmin/FirebaseAdmin/Messaging/SendResponse.cs

```diff
@@ -40,7 +40,7 @@ private SendResponse(FirebaseException exception)
         /// <summary>
         /// Gets an exception if the send operation failed. Otherwise returns null.
         /// </summary>
-        public FirebaseException Exception { get; }
+        public FirebaseMessagingException Exception { get; }
```

Comment:
I know this isn't wrong, but it also reinforces what I was saying earlier. `SendAsync` can throw non-`FirebaseMessagingException` `FirebaseException`s, but when a send is part of `SendAllAsync`, a send can only fail with a `FirebaseMessagingException`. I think the set of `Exception` types that `SendAsync` and an individual send in a `SendAllAsync` can throw should probably be the same.

Reply:
Done. FCM APIs now only throw `FirebaseMessagingException`.